cmake/cpp_bindgen.cmake.in

```diff
@@ -26,22 +26,28 @@
 #  - <library_name>_c the C-bindings with <library_name> linked to it
 #  - <library_name>_fortran the Fortran-bindings with <library_name> linked to it
 
+# TODO Do we need to protect against multiple inclusion?
+
 option(GT_ENABLE_BINDINGS_GENERATION "If turned off, bindings will not be generated." ON)
 
 # variables are unset after use for scoping, they need to be redefined in the macros
 set(__C_BINDINGS_SOURCE_DIR @__C_BINDINGS_SOURCE_DIR@)
 set(__C_BINDINGS_INCLUDE_DIR @__C_BINDINGS_INCLUDE_DIR@)
 
+add_library(cpp_bindgen_interface INTERFACE)
+target_include_directories(cpp_bindgen_interface INTERFACE ${__C_BINDINGS_INCLUDE_DIR})
+target_compile_features(cpp_bindgen_interface INTERFACE cxx_std_11)
+if(CPP_BINDGEN_GT_LEGACY)
+    target_compile_definitions(cpp_bindgen_interface INTERFACE CPP_BINDGEN_GT_LEGACY)
+endif()
+
 add_library(cpp_bindgen_generator ${__C_BINDINGS_SOURCE_DIR}/cpp_bindgen/generator.cpp)
```

Comment:
I don't remember if there was a reason... maybe you know: Why do we build the libraries in `cmake.in` That's usually supposed to be built when the project is built itself.

Reply:
Not sure I understand which part of our creative setup you don't remember:
- In your setup, e.g. COSMO, we use different compiler from C++ and Fortran, therefore it didn't make sense to build and install the Fortran modules (e.g. handle.f90) together with GridTools. We decided to install sources.
- Since we already build part of the library at the user side (handle.f90) we decided it is confusing if we build and install the corresponding C++ library (handle.cpp). Therefore nothing is build, only sources are installed.
- In GridTools the c_bindings where made available by including `bindings.cmake` which is now this file (cpp_bindgen.cmake). When we include this file we add the targets for building everything that is needed, then handle, the generator etc. The targets are then build, when the (user) project is built.
- In the standalone library, the file cpp_bindgen is already included in the cpp_bindgen-config, i.e. directly available after a find_package.

Reply:
You're right, the handle.f90 was the reason. :+1: